### PR TITLE
Use POSIX check as option to CVMFS checker

### PR DESCRIPTION
**Pull request by dberzano**
Note: done to work around broken CVMFS monitor webservice at cernvm-monitor.cern.ch during Christmas break. Do not merge yet.

### CODE REVIEW

File: check-daily-slack.py [hunk 1]
```diff
@@ -40,21 +40,29 @@ def debug(msg):
...
+    for d in getout(parrot + ["ls", "-1", "/cvmfs/alice.cern.ch"])[0].split("\n"):
+      if getout(parrot + ["bash", "-c", "ls -1 /cvmfs/alice.cern.ch/%s/Modules/modulefiles/%s/%s*" % (d, pkg, ver)])[1] == 0:
+        return True
+    # Fallback on Parrot next time
+    parrot = [ "env",
+               "HTTP_PROXY=DIRECT;",
+               "PARROT_ALLOW_SWITCHING_CVMFS_REPOSITORIES=yes",
+               "PARROT_CVMFS_REPO=<default-repositories>",
+               "parrot_run" ]
   return False
```
Comment (brinick) [inline check-daily-slack.py]: Can this not be simplified?

```python
def is_on_cvmfs_posix(pkg, tag):
   for parrot in ([], ["env", .....]):
      if getout(parrot + ["bash", "-c", "ls -1 /cvmfs/alice.cern.ch/*/Modules/....."])[1] == 0:
         return True
   return False
```

Comment (dberzano) [reply check-daily-slack.py]: Well, the `for` line would become incredibly long. This is why I did not do it that way.

Comment (brinick) [reply check-daily-slack.py]: Well, you could always put them into a variable, broken over several lines,
as in your version and then for-loop over the variable.
But anyhow, this is not very important so let me not hold you up! 😉 
It was just that when I looked at the code I was confused for some seconds
wondering what the range loop was all about.

Comment (dberzano) [reply check-daily-slack.py]: Your solution is more Pythonic (`([], long_variable)`), so I am going for it. I did not like the `range()` either to be honest, this was a dirty Christmas fix!

**Comment by dberzano** [comment]
OK, usage description added and more Pythonic syntax used.

**Comment by dberzano** [comment]
Can we merge this? I think I have addressed the comments.

**Pull request merged**